dotnet/src/InternalUtilities/planning/PlannerInstrumentation.cs

```diff
@@ -13,7 +13,7 @@ namespace Microsoft.SemanticKernel.Planning;
 #pragma warning restore IDE0130
 
 /// <summary>Surrounds the invocation of a planner with logging and metrics.</summary>
-public static class PlannerInstrumentation
+internal static class PlannerInstrumentation
```

Comment:
Does this should up in the VS Solution Explorer? 

The other classes from this folder are listed in the main solution file.